src/_ert_forward_model_runner/job.py

```diff
@@ -124,11 +131,25 @@ def ensure_file_handles_closed():
                 # state, we ignore any failures. Only seen on OSX thus far.
                 #
                 # See https://github.com/giampaolo/psutil/issues/1044#issuecomment-298745532  # noqa
-                memory = 0
-            if memory > max_memory_usage:
-                max_memory_usage = memory
+                memory_rss = 0
 
-            yield Running(self, max_memory_usage, memory)
+            max_memory_usage = max(memory_rss, max_memory_usage)
+
+            oom_score = None
+            with contextlib.suppress(ValueError, FileNotFoundError):
+                oom_score = int(
```

Comment:
This number must be the max of the oom_score for all children, otherwise it might be totally uninteresting.

Reply:
fixed.